src/components/Product.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Product.scss';
+
+class Product extends Component {
+  static propTypes = {
+    product: PropTypes.instanceOf(Object).isRequired,
+  };
+
+  render() {
+    const {
+      product,
+    } = this.props;
+
+    return (
+      <div className="onsite-product">
+        <div className="img-wrapper">
+          <a href={product.url}>
+            <img src={`https:${product.images.default}`} alt={product.name} />
```

Comment:
tem certeza que as imagens, sempre vão vir sem o protocolo, e se tiver, se vai ser https?

Eu acho que não é o caso, eu faria uma função em outro arquivo que checa se tem o protocolo, e adiciona somente se for necessário